ros/src/sensing/filters/packages/points_preprocessor/CMakeLists.txt

```diff
@@ -115,10 +123,16 @@ add_executable(points_concat_filter
         nodes/points_concat_filter/points_concat_filter.cpp
         )
 
+target_include_directories(points_concat_filter PRIVATE
+        ${PCL_INCLUDE_DIRS}
+        )
+
 target_link_libraries(points_concat_filter
         ${catkin_LIBRARIES}
         ${PCL_LIBRARIES}
+        -lyaml-cpp
```

Comment:
I think you forgot to remove this one and replace it by
`${YAML_CPP_LIBRARIES}`
Example:
https://github.com/CPFL/Autoware/blob/master/ros/src/sensing/drivers/lidar/packages/velodyne/velodyne_pointcloud/src/lib/CMakeLists.txt#L4